### PR TITLE
Remove duplicate comment in crio.conf

### PR DESCRIPTION
**Pull request by bitoku**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

It removes the duplicate comment of templateStringCrioRuntimeDefaultRuntime.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by klihub — APPROVED** [review]

**Comment by haircommander** [comment]
/approve
thank you, good find @bitoku LGTM

File: pkg/config/template.go [hunk 1]
```diff
@@ -1191,7 +1191,6 @@ const templateStringCrioRuntimeEnablePodEvents = `# Enable/disable the generatio
 `
 
 const templateStringCrioRuntimeDefaultRuntime = `# default_runtime is the _name_ of the OCI runtime to be used as the default.
-# default_runtime is the _name_ of the OCI runtime to be used as the default.
```
Comment (kwilczynski) [inline pkg/config/template.go:1194]: Nice! :+1:

**Pull request merged**